docs/operator-manual/upgrading/2.3-2.4.md

```diff
@@ -95,7 +96,10 @@ spec:
       containers:
       - name: your-plugin-name
         volumeMounts:
-        # Remove the next two lines:
         - mountPath: /tmp
-          name: tmp
+          name: tmp  # Replace this with a volume specific to your plugin, like your-plugin-name-tmp.
```

Comment:
nit but I would just `s/tmp/your-plugin-name-tmp` and remove the comment in this line